src/ConcreteOperations/minkowski_sum.jl

```diff
@@ -359,6 +362,29 @@ function _minkowski_sum_vrep_2d(vlistP::Vector{VT},
     return R
 end
 
+# assume that at least one of the arguments has length 1
+function _minkowski_sum_vrep_2d_singleton(vlistP::Vector{VT},
+                                          vlistQ::Vector{VT}) where {N, VT<:AbstractVector{N}}
+    mP = length(vlistP)
+    mQ = length(vlistQ)
+
+    if min(mP, mQ) != 1
+        throw(ArgumentError("expected one argument to have only one vertex, got $mP and $mQ respectively"))
+    elseif mQ == 1
```

Comment:
```suggestion
    elseif mP != 1
```
Otherwise you get nontermination if `mP == mQ == 1`.

Reply:
good catch!